src/vs/workbench/api/node/extHostWorkspace.ts

```diff
@@ -268,6 +268,7 @@ export class ExtHostWorkspace implements ExtHostWorkspaceShape {
 		if (folders.length === 0) {
 			return undefined;
 		}
+		// #54483 @Joh Why are we still using fsPath?
```

Comment:
@jrieken This file has still usages of `fspath`. Are they still valid and does this impact the single folder being not `file` scheme?

Reply:
This is for the deprecated `rootPath` property which is defined as file path (not uri). @sandy081 make that this returns `undefined` when the scheme isn't a `file`-uri